raft/src/test/java/org/apache/kafka/raft/RaftEventSimulationTest.java

```diff
@@ -53,7 +53,7 @@
 
 public class RaftEventSimulationTest {
     private static final int ELECTION_TIMEOUT_MS = 1000;
-    private static final int ELECTION_JITTER_MS = 0;
+    private static final int ELECTION_JITTER_MS = 10;
```

Comment:
What's the reason for bumping JITTER now?

Reply:
The only reason I didn't do it before was that I didn't have the `Random` instance wired into `KafkaRaftClient` and I wanted to keep these tests deterministic. However, in testing, I was seeing cases where elections were repeatedly being gridlocked.